README-IT.md

```diff
@@ -105,6 +106,12 @@ Se hai domande o commenti, crea un problema/issue.
 - [Le Pratiche sul COME/HOWTO rilasciare lo Software](http://en.tldp.org/HOWTO/Software-Release-Practice-HOWTO/) - Questo COME/HOWTO descrive le buone pratiche di rilascio per Linux e altri progetti open-source. Seguendo queste pratiche, renderete il più semplice possibile per gli utenti a svillupare/creare il vostro codice e usarlo, e per gli altri sviluppatori capire il vostro codice e collaborare con voi per migliorarlo.
 - [Open Sources 2.0: L'Evoluzione della Contribuzione](https://archive.org/details/opensources2.000diborich) (2005) - Open Sources 2.0 è una raccolta di saggi penetranti e stimolanti dei leader tecnologici di oggi che continua a dipingere quadro evolutivo sviluppato nel libro del 1999, Open Sources: Le voci della revoluzione.
 
+## Iniziative per contribuire all'Open Source
+- [In palio](https://up-for-grabs.net/) - un elenco di progetti con problemi per i principianti.
+- [Solo per principianti](https://www.firsttimersonly.com/) - Lista di bug etichettati come "first-timers-only".
+- [Primi contributi](https://firstcontributions.github.io/) - Fai il tuo primo contributo Open Source in 5 minuti. Uno strumento e tutorial per aiutare i principianti ad iniziare a contribuire. [Qui](https://github.com/firstcontributions/first-contributions) si trova il codice sorgente Github per il sito e l'opportunità per contribuire al repository stesso.
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - Un programma per incoraggiare i contributi Open Source. Guadagna regali come t-shirts e adesivi facendo almento 4 pull requests nel mese di ottobre.
```

Comment:
Direi "evento" invece che "programma". E i termini inglesi in italiano non vanno pluralizzati (esempio, "un bug, dei bug")
```suggestion
- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - Un evento per incoraggiare i contributi Open Source. Guadagna regali come magliette e adesivi facendo almento 4 pull request nel mese di ottobre.
```

Reply:
ciao, inanzitutto ti ringrazio. Concordo con il tuo suggerimento.

Reply:
puoi fare il "commit suggestion" qui se vuoi peer accettare il suggerimento